databricks/koalas/indexing.py

```diff
@@ -131,6 +132,89 @@ def __getitem__(self, key):
                           or len(values) > 1) else values[0]
 
 
+class iAtIndexer(_IndexerLike):
```

Comment:
Shall we name `IAtIndexer`? or we might want to rename `ILocIndexer` to `iLocIndexer`.

Reply:
pandas is using under score as prefix for the name of 'Indexers' classes like `_iAtIndexer` or `_iLocIndexer` like the below.

<img width="253" alt="스크린샷 2020-01-12 오후 2 26 55" src="https://user-images.githubusercontent.com/44108233/72214549-b5f67700-3547-11ea-876b-a5854d0324d0.png">

so, although i don't think it is not a big deal, how about we also add the under score as prefix to name of Indexers classes like they're doing???

Reply:
@ueshin 

could you check this discussion when you available??